cpp/pybind/visualization/rendering/rendering.cpp

```diff
@@ -508,51 +509,83 @@ void pybind_rendering_classes(py::module &m) {
                  "Scene.UPDATE_UV0_FLAG")
+            .def("remove_geometry", &Scene::RemoveGeometry, "name"_a,
+                 "Removes the named geometry from the scene.")
+            .def("show_geometry", &Scene::ShowGeometry, "name"_a, "show"_a,
+                 "Show or hide the named geometry.")
+            .def("geometry_is_visible", &Scene::GeometryIsVisible, "name"_a,
+                 "Returns false if the geometry is hidden, True otherwise. "
+                 "Note: this is different from whether or not the geometry is "
+                 "in view.")
+            .def("geometry_shadows", &Scene::GeometryShadows, "name"_a,
+                 "cast_shadows"_a, "receive_shadows"_a,
+                 "Controls whether an object casts and/or receives shadows: "
+                 "geometry_shadows(name, cast_shadows, receieve_shadows)")
+            .def("set_geometry_culling", &Scene::SetGeometryCulling, "name"_a,
+                 "enable"_a,
+                 "Enable/disable view frustum culling on the named object. "
+                 "Culling is enabled by default.")
+            .def("set_geometry_priority", &Scene::SetGeometryPriority, "name"_a,
+                 "priority"_a,
+                 "Set sorting priority for named object. Objects with higher "
+                 "priority will be rendering on top of overlapping geometry "
+                 "with lower priority.")
             .def("enable_indirect_light", &Scene::EnableIndirectLight,
-                 "Enables or disables indirect lighting")
-            .def("set_indirect_light", &Scene::SetIndirectLight,
+                 "enable"_a, "Enables or disables indirect lighting")
+            .def("set_indirect_light", &Scene::SetIndirectLight, "name"_a,
                  "Loads the indirect light. The name parameter is the name of "
                  "the file to load")
             .def("set_indirect_light_intensity",
-                 &Scene::SetIndirectLightIntensity,
+                 &Scene::SetIndirectLightIntensity, "intensity"_a,
                  "Sets the brightness of the indirect light")
-            .def("enable_sun_light", &Scene::EnableSunLight)
-            .def("set_sun_light", &Scene::SetSunLight,
-                 "Sets the parameters of the sun light: direction, "
+            .def("enable_sun_light", &Scene::EnableSunLight, "enable"_a)
+            .def("set_sun_light", &Scene::SetSunLight, "direction"_a, "color"_a,
+                 "intensity"_a,
+                 "Sets the parameters of the sun light direction, "
                  "color, intensity")
-            .def("add_point_light", &Scene::AddPointLight,
+            .def("add_point_light", &Scene::AddPointLight, "name"_a, "color"_a,
+                 "position"_a, "intensity"_a, "falloff"_a, "cast_shadows"_a,
                  "Adds a point light to the scene: add_point_light(name, "
                  "color, position, intensity, falloff, cast_shadows)")
```

Comment:
Can remove the function signature [`add_point_light(name, color, position, intensity, falloff, cast_shadows)` ]
from docstring - it's autogenerated if the arguments are annotated. Also in other places below.
For default values (these should follow default values in the C++ API), the syntax is `"cast_shadows"_a = false`
To document arguments:
e.g. to document `PointCloud.estimate_normals(search_param = KDTreeSearchParamKNN(), fast_normal_computation=True)`
```cpp
docstring::ClassMethodDocInject(
            m, "PointCloud", "estimate_normals",
            {{"search_param",
              "The KDTree search parameters for neighborhood search."},
             {"fast_normal_computation",
              "If true, the normal estimation uses a non-iterative method to "
              "extract the eigenvector from the covariance matrix. This is "
              "faster, but is not as numerical stable."}});
```
These appear nicely in sphinx as:
![Screenshot 2023-10-13 at 9 29 57 PM](https://github.com/isl-org/Open3D/assets/41028320/2f384ffa-4bc3-4275-89ab-4ae0532f7f53)